src/w3c/templates/show-people.js

```diff
@@ -7,19 +7,20 @@ export default (conf, name, items = []) => {
   return results;
 
   function getItem(p, i) {
+    const personName = html`${{html: p.name}}`;
     const editorid = p.w3cid ? parseInt(p.w3cid, 10): null;
     const dd = html`<dd class='p-author h-card vcard'
       data-editor-id='${editorid}'></dd>`;
       const span = document.createDocumentFragment();
     const contents = [];
     if (p.mailto) {
       contents.push(html`<a class='ed_mailto u-email email p-name'
-        href='${`mailto:${p.mailto}`}'>${p.name}</a>`);
+        href='${`mailto:${p.mailto}`}'>${personName}</a>`);
```

Comment:
We probably should just do inline `${[p.name]}`, otherwise the same text node will just change its parent three times.

Reply:
not sure I follow? Can you clarify?

Reply:
Previous ``personName = html`${{html: p.name}}` `` creates a node, ``html`${personName}` `` adds it to another parent node, and then the same ``html`${personName}` `` moves the child node to a new parent (which means the previous parent loses a child), etc.

Now `personName` is not a node but an array so it's irrelevant 😁

Reply:
Ah, yeah... got it. Whoops. Ok, yeah, so `[]` is all round better.

Reply:
See line 10 for `const personName = [p.name];`.